apps/src/templates/CodeWorkspace.jsx

```diff
@@ -39,7 +40,8 @@ class CodeWorkspace extends React.Component {
     showMakerToggle: PropTypes.bool,
     autogenerateML: PropTypes.func,
     closeWorkspaceAlert: PropTypes.func,
-    workspaceAlert: PropTypes.object
+    workspaceAlert: PropTypes.object,
+    isProjectTemplateLevel: PropTypes.bool
```

Comment:
I know you didn't choose the name for this property, but to me it suggests that the level _is_ a template (rather than that it uses one).

I feel like `hasTemplate` or `levelHasTemplate` better describes what I think this property means.

Reply:
yep I agree, I'm going to leave as-is because changing it would be pretty high risk